src/Microsoft.FeatureManagement.AspNetCore/TargetingHttpContextMiddleware.cs

```diff
@@ -18,7 +18,7 @@ public class TargetingHttpContextMiddleware
         private readonly RequestDelegate _next;
         private readonly ILogger _logger;
 
-        private const string TargetingIdKey = $"Microsoft.FeatureManagement.TargetingId";
+        private const string TargetingContextLookup = "FeatureManagement.TargetingContext";
```

Comment:
Is there any reason why we removed `Microsoft` prefix here?

Reply:
Not that I can recall... I'll add it back.